packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -755,6 +801,20 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         });
     }
 
+    private handleThemeUpdate = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+        if (this.props.onUpdateTheme) {
+            this.props.onUpdateTheme(e.target.value);
+        }
+
+        const selectedTheme: ITheme = this.props.themes.find((item: ITheme): boolean => {
+            return item.displayName === e.target.value;
```

Comment:
After you update the previous comment you would want to change item.displayName to item.id